src/posix/handlers/rename.hpp

```diff
@@ -13,8 +13,8 @@ int rename_handler(long arg0, long arg1, long arg2, long arg3, long arg4, long a
     long tid = syscall_no_intercept(SYS_gettid);
```

Comment:
Please put the `syscall_no_intrcept(SYS_gettid)` call directly in the `START_LOG` define, so that it is clear that it is compiled out when log is disabled.

Reply:
This is not possible as TID is used also inside the rename request, hence it is more logical to leave it outside